tests/tests_suite_test.go

```diff
@@ -41,10 +43,13 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 
 	managerNamespace = findManagerNamespace()
+
+	Expect(installTestsNetworkPolicy(testsNetworkPolicyName, managerNamespace)).To(Succeed())
 })
 
 var _ = AfterSuite(func() {
 	removeTestNamespaces()
+	Expect(uninstallTestsNetworkPolicy(testsNetworkPolicyName, managerNamespace)).To(Succeed())
 })
 
 var _ = JustAfterEach(func() {
```

Comment:
Can you please add in `dumpKubemacpoolLogs` func, a section logging the NetworkPolicies? It would come in handy when investigating such issues in the future.
You can use `logEndpoints` as a reference.